### ISSUE TITLE
Czech Translation

### ISSUE THREAD

**Issue by Dronf3** [katelyynn/bleh#95]
Hi, i really like bleh and i would like to translate it into Czech (my native language). 
Could i do that? lol

**Comment by katelyynn** [comment]
currently translations work alongside ur selected language in last.fm and i dont think(?) czech is an option but! in the very near future i can see adding the ability to select a language different from last.fm, which would allow this translation to work. only thing is wouldnt it be a lil jarring with the difference?

i can let u know when translation things are more stabilised thou if ur interested :3 once this big update is released it should be mostly fine.

**Comment by Dronf3** [comment]
Yeah, Czech sadly isn't an option in last.fm. Although, do let me know when the translation could be possible.

> only thing is wouldnt it be a lil jarring with the difference?

I'm not sure what you mean by that though..

**Comment by katelyynn** [comment]
> > only thing is wouldnt it be a lil jarring with the difference?
> 
> I'm not sure what you mean by that though..

ohh all i meant is wouldnt the difference between last.fm being in english and bleh being in czech be a little strange
